### DISCUSSION TITLE
Uploading non English paperwork as proof (how to handle translation)

### DISCUSSION THREAD

**Discussion by ZionLG** [community/community#75623] [GitHub Education]
I want to upload my proof for studying at school, but the paperwork is not in English. When I took a picture and added the English translation next to the original text, GitHub blocked it saying "the picture seems edited", when I try to take a picture of it from phone, it tells me it cant find the school name (as its not in English) and i need to add translation next to the original photo, so how am I supposed to do it?

**Comment by elderguardian** [comment]
I think you should contact the support about this
https://support.github.com/

**Comment by wasimtikki120** [comment]
Uploading non-English documents or images on GitHub, especially when they contain translations or additional text, can sometimes trigger automated filters designed to detect potential issues like inappropriate content or spam. To address this, you can take a few steps to ensure your proof is successfully uploaded:

1. **Separate Translation and Original Image:**
   - Instead of placing the English translation directly next to the original text in the same image, try uploading two separate images. One image could contain the original non-English text, and the other image could include the English translation.

2. **Provide Context in README or Comments:**
   - Create a README file in the repository where you intend to upload the proof. In the README file or in comments, provide context about the images you're uploading. Clearly explain that one image contains the original non-English text, and the other contains the English translation.

3. **Explicitly Mention Translation in Commit Message:**
   - When you commit and push your changes to GitHub, use a commit message that explicitly mentions the purpose of the images. For example, you might say, "Added images: Original non-English text and English translation for school proof."

4. **Contact GitHub Support:**
   - If the issue persists, consider reaching out to GitHub Support for assistance. You can describe your situation and explain that you are trying to upload non-English documents with translations for a legitimate purpose. GitHub Support may be able to provide further guidance or assistance.

5. **Consider Alternatives:**
   - If GitHub's platform continues to present challenges for your specific use case, consider exploring alternative platforms for document sharing. Platforms like Google Drive, Dropbox, or even a personal website could be used to share your proof with others.

Remember, GitHub's automated filters are in place to maintain a positive and secure environment for all users. By providing clear context and separating the original text and its translation, you increase the likelihood of successfully sharing your proof without triggering false positives in the automated filters.

**Comment by ZionLG** [comment]
Tried that. the support subjects ("What can GitHub Education assist you
with?") regarding github education have nothing to do with my problem or
send me to the community and dont allow me to open a ticket

‫בתאריך יום ד׳, 15 בנוב׳ 2023 ב-19:34 מאת ‪Luca‬‏ ***@***.***
‬‏>:‬
